src/Scheme/Evaluator.hs

```
@@ -27,11 +27,11 @@ eval val@(Number _) = return val
 eval val@(Bool _) = return val
 eval (Atom id) = getVar id
 eval (List [Atom "quote", val]) = return val
--- FIXME: Make alt optional
-eval (List [Atom "if", pred, conseq, alt]) = do
+eval (List (Atom "if" : pred : conseq : varargs)) = do
     result <- eval pred
-    case result of
-        Bool False -> eval alt
+    case (result, varargs) of
```

You should throw an error if `varargs` has more than one elements.